corehq/apps/locations/views.py

```diff
@@ -389,6 +390,17 @@ def unique_name_and_code():
                         return False
             return True
 
+        def _verify_has_users_config(loc_type_payload, pk):
+            if not loc_type.get('has_users'):
+                try:
+                    existing_loc_type_sql_obj = LocationType.objects.get(pk=pk)
+                except ObjectDoesNotExist:  # Location type not created yet
```

Comment:
Looks like there's also a `_is_fake_pk` bit of logic that you can use to avoid looking up uncreated location types.